# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/firefox/no-jit.diff
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- firefox-99.0.1/js/moz.configure.vanilla	2022-06-14 17:33:22.592134685 +0200
+++ firefox-99.0.1/js/moz.configure	2022-06-14 17:41:11.544106733 +0200
@@ -1257,8 +1257,8 @@
 
 @depends("--enable-simulator", target)
 def wasm_has_heapreg(simulator, target):
-    if target.cpu != "x86":
-        return True
+    #if target.cpu != "x86":
+    return True
 
     if simulator and simulator[0] == "arm":
         return True
